src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -114,10 +114,48 @@
             "$ref": "base.definition.schema.v1.json#/definitions/httpsUrl"
         },
         "taggable": {
-            "description": "A boolean flag indicating whether this resource type supports tagging.",
+            "description": "(deprecated) A boolean flag indicating whether this resource type supports tagging.",
             "type": "boolean",
             "default": true
         },
+        "tagging": {
+            "type": "object",
+            "properties": {
+                "taggable": {
+                    "description": "A boolean flag indicating whether this resource type supports tagging.",
+                    "type": "boolean",
+                    "default": true
+                },
+                "tagOnCreate": {
+                    "description": "A boolean flag indicating whether this resource type supports tagging resources upon creation.",
+                    "type": "boolean",
+                    "default": true
+                },
+                "tagUpdatable": {
+                    "description": "A boolean flag indicating whether this resource type supports updatable tagging.",
+                    "type": "boolean",
+                    "default": true
+                },
+                "CFNSystemTags": {
```

Comment:
`CFNSystemTags`?  Don't use short abbr. for public name.  may `cloudFormationSystemTags`?

Reply:
sure, will change it